eagleye_pp/eagleye_pp/src/eagleye_pp_output.cpp

```diff
@@ -302,19 +302,23 @@ void eagleye_pp::writeDetailCSVOneWay(std::ofstream* output_log_csv_file, const
 ,velocity.twist.linear.x,velocity.twist.linear.y,velocity.twist.linear.z,velocity.twist.angular.x,velocity.twist.angular.y,velocity.twist.angular.z\
 ,velocity_scale_factor.scale_factor,correction_velocity.twist.linear.x,correction_velocity.twist.linear.y,correction_velocity.twist.linear.z,correction_velocity.twist.angular.x,correction_velocity.twist.angular.y,correction_velocity.twist.angular.z,velocity_scale_factor.status.enabled_status,velocity_scale_factor.status.estimate_status\
 ,distance.distance,distance.status.enabled_status,distance.status.estimate_status\
-,heading_1st.heading_angle,heading_1st.status.enabled_status,heading_1st.status.estimate_status\
-,heading_interpolate_1st.heading_angle,heading_interpolate_1st.status.enabled_status,heading_interpolate_1st.status.estimate_status\
-,heading_2nd.heading_angle,heading_2nd.status.enabled_status,heading_2nd.status.estimate_status\
-,heading_interpolate_2nd.heading_angle,heading_interpolate_2nd.status.enabled_status,heading_interpolate_2nd.status.estimate_status\
-,heading_3rd.heading_angle,heading_3rd.status.enabled_status,heading_3rd.status.estimate_status\
-,heading_interpolate_3rd.heading_angle,heading_interpolate_3rd.status.enabled_status,heading_interpolate_3rd.status.estimate_status\
+,heading_1st.heading_angle,heading_1st.variance,heading_1st.status.enabled_status,heading_1st.status.estimate_status\
```

Comment:
trajectory_plot/eagleye_pp_single_evaluation.py, which changes the log file, will stop working and must be addressed.

Reply:
Same here.
https://github.com/MapIV/eagleye/blob/94db195a9edc35bb00265e60334eb56087b59dc6/eagleye_rt/src/monitor_node.cpp#L873

Reply:
I found that eagleye_pp_single_evaluation.py is fine because it specifies the name, not the column number